src/openfermion/ops/_fermion_operator.py

```diff
@@ -205,3 +205,75 @@ def is_molecular_term(self):
             if not (particles == spin == 0):
                 return False
         return True
+
+    def freeze_orbitals(self, occupied, unoccupied=[], prune=True):
+        """Fix some orbitals to be occupied and others unoccupied.
+
+        Removes all operators acting on the specified orbitals, and flips the
+        sign of each term as appropriate. Terms whose expectation value
+        between states with the specified occupancy is zero are removed.
+        Additionally, the indices are renumbered such that the frozen orbitals
+        are eliminated.
+
+        Note that the sign change accounts for both the reordering of operators
+        needed to group the operators acting on the frozen orbitals, as well
+        as the impact of pruning frozen orbitals on the Jordan-Wigner strings
```

Comment:
As a matter of phrasing, I would say these things result from fermion anti-commutation rather than the JW representation of that.

Reply:
I've reworded the comments to (hopefully) be more clear and avoid mentioning JW strings.